src/ingestor.js

```diff
@@ -10,44 +10,67 @@
  * governing permissions and limitations under the License.
  */
 
-import { fetch as originalFetch } from '@adobe/fetch';
-import { forEachLimit, mapLimit } from 'async';
-import fetchBuilder from 'fetch-retry';
+import { FetchRetry } from '@adobe/content-lake-commons';
```

Comment:
I haven't looked at the commons bundle yet - did we modify `fetch-retry` in that package?

Reply:
Yep, essentially the retry logic was pulled out of here into commons.